ui/components/src/ui-components/propertygrid/dataproviders/filterers/PropertyDataFiltererBase.ts

```diff
@@ -64,5 +66,6 @@ export interface IPropertyDataFilterer {
 export abstract class PropertyDataFiltererBase implements IPropertyDataFilterer {
   public onFilterChanged: PropertyFilterChangeEvent = new PropertyFilterChangeEvent();
   public abstract get isActive(): boolean;
-  public abstract async matchesFilter(node: PropertyRecord, parents: PropertyRecord[]): Promise<PropertyDataFilterResult>;
+  public abstract async recordMatchesFilter(node: PropertyRecord, parents: PropertyRecord[]): Promise<PropertyDataFilterResult>;
```

Comment:
maybe it would make sense to additionally have:
- `PropertyRecordDataFiltererBase` which implements `categoryMatchesFilter` with `{return !this.isActive;}`
- `PropertyCategoryDataFiltererBase` which implements `recordMatchesFilter` with `{return !this.isActive;}`
then concrete filterers wouldn't have to implement these methods..

Reply:
Created these bases